config/initializers/rack_attack.rb

```diff
@@ -58,13 +58,13 @@ def remote_ip
   # Always allow requests from below IP addresses for load testing
   # `100.103.248.0 - 100.103.248.255`
   # `100.103.251.128 - 100.103.251.255`
-  # `10.247.104.23`
+  # `10.247.104.` - tevi-dev-load-testing host IPs
   # (blocklist & throttles are skipped)
   Rack::Attack.safelist('allow requests from loadtest host') do |req|
     # Requests are allowed if the return value is truthy
     req.ip.match?(/100.103.248.(\b[0-9]\b|\b[1-9][0-9]\b|1[0-9]{2}|2[0-4][0-9]|25[0-5])
                   |100.103.251.(12[8-9]|1[3-9]\d|2[0-4]\d|25[0-5])
-                  |10.247.104.23/)
+                  |10.247.104./)
```

Comment:
Are you whitelisting all hosts from 10.247.104.*, do you need the .* here?

Reply:
I see that you were provided with a [range of IPs](https://github.com/department-of-veterans-affairs/va.gov-team/issues/42465#issuecomment-1151333657), but just want to check on the syntax here.

Reply:
Yes @LindseySaari, we are using a couple of EC2 instances with 10.247.104.* IP to generate load. So we need .* here to avoid opening multiple PRs for each host

Reply:
@kanchanasuriya seems like you missed a  `.*` in the end, isn't that needed?

Reply:
It is not needed @randomsync, IP will still be in safelist